integrations/call911/server/server.ts

```diff
@@ -27,6 +27,12 @@ onNet(Events.Call911ToServer, async ({ street, name, description, position }: an
     location: street,
     description: description.join(" "),
     postal,
+    gtaMapPosition: {
+        x: position.x,
+        y: position.y,
+        z: position.z,
+        heading: 0,
```

Comment:
Maybe this could be implemented: https://docs.fivem.net/natives/?_0xE83D4F9BA2A38914, if you're not sure how to, don't worry